client/src/components/UploadField/UploadFieldItem.js

```diff
@@ -226,7 +226,7 @@ class UploadFieldItem extends Component {
 
     if (message !== null) {
       return (
-        <div className="uploadfield-item__error-message">
+        <div className="uploadfield-item__error-message" title="{message}">
```

Comment:
Actually, I think it needs to be this:
```suggestion
        <div className="uploadfield-item__error-message" title={message}>
```